app/controllers/clone_pages_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ClonePagesController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    page
```

Comment:
the pattern we usually use for this is to put a before filter that calls the `assign_page` method or something, then you just use `@page` freely in the action.